config/webpack.target.services.js

```diff
@@ -44,6 +44,7 @@ module.exports = {
     filename: '[name].js'
   },
   optimization: {
+    minimize: false,
```

Comment:
Why this?

Reply:
Because it's easier to understand stack trace, and it is speedier to build, this should not be commmited. I am fixing right now.